### ISSUE TITLE
Is the CEC firmware open source?

### ISSUE THREAD

**Issue by bubnikv** [phdlee/uvk5cec#74]
Hello KD8CEC

I wonder whether your firmware is still open source? It seems your last commit was done end of January, however in release section you are releasing modifications continuously without a source code. 

Did I overlook anything or did you decide to continue with a closed source development?

Thanks and VY 73,
Vojtech OK1IAK

**Comment by bubnikv** [comment]
> As announced in advance, various experimental versions will be released up to V1.0, and the source code will be registered when V1.0 is released. Until V1.0, we will only upload firmware binary files.

I see. What is the reason you decided to keep from opening the source code?

**Comment by nicolalamacchia** [comment]
No, it's not open source, that's why I migrated to https://github.com/armel/uv-k5-firmware-custom, which is actively maintained. It lacks CW, but I might look into that in the future and maybe try to make a PR for it myself, if @armel's doesn't implement it in the meantime :)

**Comment by tristanmoney** [comment]
There are features here that no other firmware has. FT8, WSPR, APRS, CW, etc. Last time Ian went quiet for awhile he came back with the C-board and a great UI. I think he is probably trying to focus on development and will come back eventually and release the source code.

**Comment by mlforcada** [comment]
OK, so 0.3 is not free/open-source, which is very unfortunate. What keeps the developer from letting us see the code?. What is exactly the last version with source code?
